src/mlpack/core/optimizers/cmaes/random.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <ctime>
+#include <cmath>
+
+/**
+ * @class Random
+ * A pseudo random number generator.
+ */
+template<typename T>
+class Random
```

Comment:
dont know how it happened this time.
the randomNormal() works fine, not even Random or randu().
glad that at last it got solved.
thanks @zoq 
the changes in seed is very well getting reflected if a user wants.
this took a lot of time. 
during all these days also zoq i worked as i could just that things were creating trouble.
im starting with the ANN code. Lets see how it goes
thanks again :)